modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/CallrailAdapter.scala

```diff
@@ -37,19 +32,14 @@ import Adapter.Adapted
  * a known version of the AD-X Tracking webhook
  * into raw events.
  */
-object CallrailAdapter extends Adapter {
+class CallrailAdapter(config: CallrailConfig) extends Adapter {
```

Comment:
```suggestion
case class CallrailAdapter(config: CallrailConfig) extends Adapter {
```

Reply:
Have changed all adapters to `case class`